dbt/adapters/athena/lakeformation.py

```diff
@@ -75,18 +89,26 @@ def _remove_lf_tags_columns(self, existing_lf_tags: GetResourceLFTagsResponseTyp
                     )
                     self._parse_and_log_lf_response(response, columns, {tag_key: tag_value}, "remove")
 
+    @staticmethod
+    def _table_tags_to_remove(
+        lf_tags_table: list[LFTagPairTypeDef], lf_tags: Optional[Dict[str, str]], lf_inherited_tags: set[str]
```

Comment:
```suggestion
        lf_tags_table: list[LFTagPairTypeDef], lf_tags: Optional[Dict[str, str]], lf_inherited_tags: Set[str]
```
seems unit tests are failed because of that

Reply:
Thanks I'm new to type annotating Python

Reply:
@svdimchenko Regarding unit tests, I didn't have a failure previously and don't know. Are you seeing the `xfail` expected failure, maybe? That tripped me up at first.